### PR TITLE
Ruby and nuget `readme.md` link to documentation site

### LINKED ISSUE #494: Improve package readme pages
Add better user facing information to our README pages that show up in package manager web sites, such as https://www.npmjs.com/package/@trinsic/trinsic-web and https://www.nuget.org/packages/Trinsic/

Languages:
- [x] .NET https://github.com/trinsic-id/sdk/pull/539
- [x] ~Go~ (source code direct)
- [x] ~Java~ (no central repo)
- [x] Node https://github.com/trinsic-id/sdk/pull/539
- [x] Python https://github.com/trinsic-id/sdk/pull/538
- [x] Ruby https://github.com/trinsic-id/sdk/pull/539
- [x] ~Swift~ (source code direct)
- [x] Web https://github.com/trinsic-id/sdk/pull/539

### PR DESCRIPTION
**Pull request by fundthmcalculus**
closes #494

### CODE REVIEW

**Comment by michaeldboyd** [comment]
Blocked on final test 
image

**Comment by fundthmcalculus** [comment]
@michaeldboyd typescript web failure is a separate issue with chromedriver github action updates. I will fix that in a separate PR.

File: ruby/trinsic.gemspec [hunk 1]
```diff
@@ -7,6 +7,7 @@ Gem::Specification.new do |spec|
   spec.email         = ['polygonguru@gmail.com']
 
   spec.summary       = 'Trinsic Services Ruby SDK'
+  spec.description   = 'Ruby language wrapper for the Trinsic services. Documentation site is here: https://docs-v2.trinsic.id/'
```
Comment (geel9) [inline ruby/trinsic.gemspec:10]: Personal style opinion: I might change the last bit of this to "Documentation [available] at {url}" or "View documentation at {url}".

**Review by geel9 — APPROVED** [review]
Looks good to me. Test failure unrelated and fixed in another PR.

**Pull request merged**